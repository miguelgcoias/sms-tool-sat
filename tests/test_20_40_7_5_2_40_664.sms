20
28 5 33 1 5
5 2 9 2 1 1
19 5 29 4 1 1 1 2
33 6 39 2 3 3
26 6 32 1 6
0 6 6 3 2 3 1
5 5 15 1 5
1 2 5 1 2
12 7 26 4 1 2 1 3
13 6 25 3 3 2 1
19 4 27 1 4
7 7 21 2 4 3
25 7 39 2 4 3
22 1 24 1 1
4 1 5 1 1
15 1 17 1 1
25 1 26 1 1
34 7 41 5 1 1 1 3 1
13 6 25 4 2 2 1 1
30 6 36 5 1 1 1 2 1
2 17 11
0
4 19 14 10 9
5 6 12 2 8 17
6 15 16 10 2 7 19
0
2 15 6
0
2 6 12
2 8 6
5 9 12 10 19 6
3 8 15 6
5 9 2 17 7 3
2 16 15
1 8
4 6 12 2 15
5 16 7 19 11 3
5 5 19 15 3 2
3 15 6 2
4 15 2 19 14
