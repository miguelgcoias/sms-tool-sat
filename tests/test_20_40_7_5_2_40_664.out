3
2 5 6
8 1
15 4
